File: QA-typeORM/QuestionAnswer/src/entity/Questions.ts
import {Entity, PrimaryGeneratedColumn, Column, OneToMany, ManyToOne, JoinColumn} from "typeorm";
import { Answers } from "./Answers";
import { Users } from "./Users";

@Entity()
export class Questions {

    @PrimaryGeneratedColumn()
    id: number;

    @Column()
    description: string;

    @ManyToOne(type => Users, users => users.questions)
    @JoinColumn({name: "user_id"})
    users: Users[]

    @OneToMany(type => Answers, answers => answers.questions)
    answers: Answers[]

}

File: QA-hibernate/qa-hibernate/src/main/java/com/example/qahibernate/entities/Questions.java
package com.example.qahibernate.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
/**
 * Questions
 */
@Entity(name="questions")
public class Questions {

    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private Long id;

    @Column(name = "description")
    private String description;

    @Column(name = "user_id")
    private long userId;



    public Long getId() {
        return this.id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getDescription() {
        return this.description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public long getUserId() {
        return this.userId;
    }

    public void setUserId(long userId) {
        this.userId = userId;
    }


}